src/components/IngenData.tsx

```diff
@@ -0,0 +1,15 @@
+import { AlertStripeFeil } from 'nav-frontend-alertstriper';
+import React from 'react';
+
+const IngenData = () => {
+  return (
+    <div className='limit'>
```

Comment:
Ordet "limit" gir ingen mening for dette komponentet. Burde hete noe med "ingendata" e.l

Reply:
> Ordet "limit" gir ingen mening for dette komponentet. Burde hete noe med "ingendata" e.l

Enig, det er tatt fra Sporenstreks, da bør vi endre der også